platform/lib/samples/README.md

```diff
@@ -0,0 +1,43 @@
+# AMP by Example API
+
+Generates embeddable AMP source code snippets with previews. The input format follows the same rules as described [here](../README.md).
+
+Example:
+
+```
+const abe = require('amp-by-example');
```

Comment:
This is also no longer valid.
I think we can reduce (remove?) this readme, since we use this is now used only internally in a specific way.

Reply:
done